internal/pkg/describe/status.go

```diff
@@ -401,3 +435,52 @@ func statusColor(status string) string {
 		return color.Red.Sprint(status)
 	}
 }
+
+type targetHealthStatus struct {
+	TargetHealth elbv2.TargetHealth `json:""`
```

Comment:
Why do we have "" for `TargetHealth`?

Reply:
Updated! It is `json:"healthDescription"` now.